test/bigquery.test.ts

```diff
@@ -114,10 +114,11 @@ describe('BigQueryFormatter', () => {
     `);
   });
 
+  // TODO: Possibly incorrect formatting of STRUCT<>() and ARRAY<>()
```

Comment:
This is currently an unfortunate change in formatting.

This whole typed arrays/structs parsing needs a complete rewrite though. The current solution is a big hack. It should really be parsed inside Parser not hacked together inside postProcess() of tokenizer.